app/components/application/errors/application-error-display.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import { autobind } from "core-decorators";
+
+import { SidebarManager } from "app/components/base/sidebar";
+import { AccountResource, Application } from "app/models";
+import { AccountService } from "app/services";
+
+@Component({
+    selector: "bex-application-error-display",
+    templateUrl: "application-error-display.html",
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ApplicationErrorDisplayComponent {
+    @Input()
+    public application: Application;
+
+    private _batchAccount: AccountResource;
+
+    constructor(
+        private accountService: AccountService,
+        private sidebarManager: SidebarManager) {
+
+        accountService.currentAccount.subscribe((account) => {
+            this._batchAccount = account;
+        });
+    }
+
+    public get hasLinkedStorageAccountIssue(): boolean {
+        if (this._batchAccount && this._batchAccount.properties) {
+            return !Boolean(this._batchAccount.properties.autoStorage)
```

Comment:
here you don't need to cast to boolean if you do ! it will do it by itself.

`Boolean` is instead of double negation `!!`